### DISCUSSION TITLE
v0.12.0

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/steamguard-cli#312] [General]
## What's Changed
* adjust error message for when mobile confirmation fails by @dyc3 in https://github.com/dyc3/steamguard-cli/pull/311

**Full Changelog**: https://github.com/dyc3/steamguard-cli/compare/v0.11.0...v0.12.0

<hr /><em>This discussion was created from the release <a href='https://github.com/dyc3/steamguard-cli/releases/tag/v0.12.0'>v0.12.0</a>.</em>

**Comment by Darkinvo** [comment]
it is not install in linux :
$ sudo dpkg -i steamguard-cli_0.12.0-0.deb  
    
`dpkg-deb: error: archive 'steamguard-cli_0.12.0-0.deb' uses unknown compression for member 'control.tar.zst', giving up`
`dpkg: error processing archive steamguard-cli_0.12.0-0.deb (--install):`
` dpkg-deb --control subprocess returned error exit status 2`
`Errors were encountered while processing:`
` steamguard-cli_0.12.0-0.deb`
